.github/workflows/dotnet-format.yml

```diff
@@ -0,0 +1,21 @@
+name: .NET format
+
+on:
+  push:
+    branches: [ "main" ]
+  pull_request:
+    branches: [ "main" ]
+
+jobs:
+  dotnet-format:
+
+    runs-on: windows-latest
```

Comment:
Just wondering why we run on Windows :P

Reply:
If run on Linux, there is a lot of "violations" related to end-of-line stuff - which we can't locally fix as we're all using Windows.